data/Language/Python.yaml

```diff
@@ -16,3 +16,36 @@ delimiters:
   - square_braces
   - curly_braces
   - colon_indent
+keywords:
+  - and
+  - as
+  - assert
+  - break
+  - class
+  - continue
+  - def
+  - del
+  - elif
+  - else
+  - except
+  - False
```

Comment:
needs single quotes.

See it is missing from the list on https://deploy-preview-18--coast.netlify.com/python/